tests/integration/operator/deploy_utils.go

```diff
@@ -0,0 +1,198 @@
+// Copyright Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package integrationoperator
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+
+	. "github.com/onsi/ginkgo/v2"
+	"maistra.io/istio-operator/api/v1alpha1"
+	"maistra.io/istio-operator/pkg/util/tests/helm"
+	"maistra.io/istio-operator/pkg/util/tests/kubectl"
+	"sigs.k8s.io/yaml"
+)
+
+type Action int
+
+const (
+	Delete Action = iota
+	Apply
+	Deploy
+	Undeploy
+)
+
```

Comment:
These constants aren't used anywhere. It's weird that the linter doesn't catch this, because I know that it detects unused code. Are we sure that the linter isn't skipping this file?

Reply:
Let me check this

Reply:
Ah, it's because these constants are exported. If these were lowercase, the linter would complain.